### ISSUE TITLE
MQTT over WebSockets

### ISSUE THREAD

**Issue by leandronsp** [eclipse/paho.mqtt.ruby#8]
Hey!

Is it planned some support for WebSockets?

**Comment by p-goudet** [comment]
@leandronsp 

I agree that it would be a nice feature to a add. I think it would be include in the future, unfortunately I can't tell you a precise ETA, as it is not planned for the next version.

**Comment by leandronsp** [comment]
Nice, thanks, I'll try to submit some contribution in that regard, will update soon!
